src/Illuminate/Support/Number.php

```diff
@@ -295,6 +295,20 @@ public static function useLocale(string $locale)
         static::$locale = $locale;
     }
 
+    /**
+     * Returns a random number, using mt_rand() function, using the given length.
+     *
+     * @param  int  $length
+     * @return int
+     */
+    public static function random(int $length = 16): int
+    {
+        $min = (int) ('1'.str_repeat('0', $length - 1)); // eq. 100000
+        $max = (int) str_repeat('9', $length); // eq. 999999
```

Comment:
```suggestion
        $min = 10 ** ($length - 1); // eq: 100000
        $max = (10 ** $length) - 1; // eq. 999999
```

This would be an approach without using strings, and only mathematical operations.

Reply:
If that works, also fine by me. I just missed the Number::random() method :-D